wavefront/system_stats.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2018 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// nolint:lll
+// Generates the wavefront adapter's resource yaml. It contains the adapter's
+// configuration, name, supported template names (metric in this case), and
+// whether it is session or no-session based.
+
+package wavefront
+
+import (
+	"runtime"
+	"time"
+
+	"github.com/mackerelio/go-osstat/cpu"
+	metrics "github.com/rcrowley/go-metrics"
+	wf "github.com/wavefrontHQ/go-metrics-wavefront"
+	"istio.io/istio/pkg/log"
+)
+
+const tickerDuration = 5 * time.Second
+
+var before *cpu.Stats
+var after *cpu.Stats
+var err error
+
+// CreateSystemStatsReporter creates a reporter that periodically flushes adpater system metrics to Wavefront.
+func CreateSystemStatsReporter(hostTags map[string]string) {
+	log.Info("Preparing adapter metrics")
```

Comment:
nit: lower-case Preparing -> preparing, and other logs below (just to keep it consistent with the other statements)